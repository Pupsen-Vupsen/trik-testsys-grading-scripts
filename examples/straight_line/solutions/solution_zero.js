var __interpretation_started_timestamp__;
var pi = 3.141592653589793;

var main = function()
{
	__interpretation_started_timestamp__ = Date.now();
	
	brick.motor(M3).setPower(100);
	brick.motor(M4).setPower(100);
	
	script.wait(1250);
	
	brick.motor(M1).brake();
	brick.motor(M2).brake();
	brick.motor(M3).brake();
	brick.motor(M4).brake();
	
	return;
}
