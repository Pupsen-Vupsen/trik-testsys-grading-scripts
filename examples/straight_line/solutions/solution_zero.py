import sys
import time
import random
import math

class Program():
  __interpretation_started_timestamp__ = time.time() * 1000

  pi = 3.141592653589793

  def execMain(self):

    
    brick.motor("M3").setPower(100)
    brick.motor("M4").setPower(100)
    
    script.wait(1250)
    
    brick.motor("M1").brake()
    brick.motor("M2").brake()
    brick.motor("M3").brake()
    brick.motor("M4").brake()
    
    brick.stop()
    return

def main():
  program = Program()
  program.execMain()

if __name__ == '__main__':
  main()
