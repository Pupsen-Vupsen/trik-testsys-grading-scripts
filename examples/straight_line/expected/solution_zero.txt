-------- Results for ./results/straight_line/solution_zero --------
./results/straight_line/solution_zero/1_square.json: ERROR
./results/straight_line/solution_zero/2_square.json: ERROR
TOTAL: 0
